assets/app/view/game/part/assignments.rb

```diff
@@ -16,16 +16,24 @@ class Assignments < Base
         DELTA = (ICON_RADIUS * 2) + 2
 
         def preferred_render_locations
+          unless @assignments.nil?
+            assignments_to_show = @assignments.reject { |_key, value| value == Engine::Assignable::STACKABLE }
+            stack_count = @assignments.size - assignments_to_show.size
+            if stack_count.positive?
+              assignments_to_show[@assignments.key(Engine::Assignable::STACKABLE)] =
+                Engine::Assignable::STACKABLE
+            end
```

Comment:
If I'm reading correctly, this code seems to assume there is only one stackable type on a hex. If so, this should be more generic and each type of icon should be separately stackable.

Reply:
Updated to be able to support mulitple stacks